importer/v8/importer.go

```diff
@@ -146,7 +162,7 @@ func (i *Importer) processDML(scanner *bufio.Scanner) {
 		if strings.HasPrefix(line, "#") {
 			continue
 		}
-		i.batchAccumulator(line)
+		i.batchAccumulator(line, start)
 	}
```

Comment:
This isn't cosmetic.  It has to be at the beginning before the loop or it won't be able to calculate the total time lapsed.